libcodechecker/libhandlers/analyzers.py

```diff
@@ -86,6 +106,17 @@ def main(args):
             analyzer_types.supported_analyzers,
             context)
 
+    if args.dump_config:
+        binary = context.analyzer_binaries.get(args.dump_config)
+
+        if args.dump_config == 'clang-tidy':
+            subprocess.call([binary, '-dump-config', '-checks=*'])
+        elif args.dump_config == 'clangsa':
+            # TODO: Not supported by ClangSA yet!
+            LOG.warning("'--dump-config clangsa' is not supported yet.")
```

Comment:
For me this message was never be printed.
I checked the [logger.conf](https://github.com/Ericsson/codechecker/blob/master/config/logger.conf) file and it doesn't contain a `logger` for `analyze` type:
https://github.com/Ericsson/codechecker/blob/0df57dcf8fa83bc5cb4fac5a779bd109d6b2d32d/libcodechecker/libhandlers/analyzers.py#L22